tools/donate-cpu.py

```diff
@@ -168,19 +168,16 @@ def wget(url, destfile, bandwidth_limit):
         print('wget failed with ' + str(exitcode))
         os.remove(destfile)
         return False
-    if os.path.isfile(destfile):
-        return True
-    print('Sleep for 10 seconds..')
-    time.sleep(10)
```

Comment:
I guess the sleep() was here to wait a little bit until trying wget again. Sometimes the problem is gone after a short pause.

Reply:
Same as the other comment `--tries` should take care of it and it will also never reach this code as even if the download fails the file will exist although with 0 bytes. I left just in case there just to be safe.

Reply:
Yes, `--tries` really should be enough. Then the changes are ok.